server/src/main/java/org/candlepin/dto/api/v1/ConsumerDTO.java

```diff
@@ -1033,6 +1092,12 @@ public ConsumerDTO clone() {
         copy.setCapabilities(this.getCapabilities());
         copy.setContentTags(this.getContentTags());
 
+        copy.setServiceLevel(this.getServiceLevel());
+        copy.setRole(this.getRole());
+        copy.setUsage(this.getUsage());
+        copy.setAddOns(this.getAddOns());
+        copy.setSystemPurposeStatus(this.getSystemPurposeStatus());
```

Comment:
No need to explicitly assign Strings like role, usage, servicelevel, systempurposestatus etc. in the clone method, since they're already on the copy when `ConsumerDTO copy = super.clone();` is called, and they're immutable, so they can be safely shared between the original and the cloned objects. Only setAddons() is required here because its a collection.

Reply:
Done. Will be included in next update of this PR. Thanks!